pipes/WDL/tasks/tasks_reports.wdl

```diff
@@ -464,6 +464,7 @@ task MultiQC {
     Array[String]? module_to_use
     Boolean        data_dir = false
     Boolean        no_data_dir = false
+    Boolean        show_sample_names_with_paths = false
```

Comment:
Is there a reason why we’d actually want to support the —dirs argument in WDL? We could probably drop this parameter

Reply:
Theoretically to disambiguate same-name samples

Reply:
If the "sample name" (without the `--dirs` argument) is simply the basename of the filename, I think we already did some work earlier to ensure that our demux task was emitting unique filenames for everything (e.g. appending lane numbers and such to the filename so that our nextseq output files were all uniquely named)